aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,336 @@
+
+If you remove the ".fr" culture designator AND you have the culture set to French (via cookie or other mechanism), the default resource file is read and strings are localized. This is expected behavior. \*.resx designates a default or fallback resource, so that when nothing meets your requested culture you're served \*.resx. If you want to just return the key when missing a resource for the requested culture you must not have a default resource file.
+
+
+Adding Other Cultures  
+-----------------------
+ 
+Each language and culture combination (other than the default language) requires a unique resource file. You can create resource files for different cultures and locales by creating new resource files in which the ISO language codes are part of the file name (for example, **en-us**, **fr-ca**, and **en-gb**). These ISO codes are placed between the file name and the .resx file name extension, as in *Welcome.es-MX.resx* (Spanish/Mexico). To specify a culturally neutral language, you would eliminate the country code, such as Welcome.fr.resx for the French language. 
+
+
+Preparing your app for localization
+----------------------------------------
+
+Introduced in ASP.NET Core, ``IStringLocalizer`` was architected to improve productivity when developing localized apps. The simple interface has an indexer and an ``IEnumerable`` for returning localized strings. ``IStringLocalizer`` doesn't require you to store the default language strings in a resource file. You can develop an app targeted for localization and not need to create resource files early in development. We anticipate that many developers will **not** create a default language resource file. The code below shows how to wrap the string "About Title" for localization.
+
+.. literalinclude:: localization/sample/Controllers/AboutController.cs
+  :language: c#
+
+In the code above, the ``IStringLocalizer`` implementation comes from :doc:`/fundamentals/dependency-injection`. I'll show how the ``IStringLocalizer`` service gets added in the **Configuring localization** section below. If the localized value of "About Title" is not found, then the indexer key is returned, that is, the string "About Title". You can leave the default language literal strings in the app and wrap them in the localizer. You can develop your app with your default language and prepare it for the localization step without first creating a default resource file. Alternatively, you can use the traditional approach and provide a key to retrieve the default language string. For many developers the new workflow of not having a default language *.resx* file and simply wrapping the string literals can reduce the overhead of localizing an app. Other developers will prefer the traditional work flow as it can make it easier to work with longer string literals and make it easier to update localized strings.
```

Comment:
Maybe `...leave the default language literal strings in the app and wrap them in the localizer, so that you can develop your app`. The two concepts seem connected enough that they feel strange to me as separate sentences.